target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/10-ath9k-eeprom

```diff
@@ -56,6 +56,10 @@ case "$FIRMWARE" in
 		caldata_extract "calibrate" 0x1000 0x440
 		ath9k_patch_mac $(mtd_get_mac_ascii u-boot-env0 RADIOADDR1)
 		;;
+	extreme,ap3805i)
+		caldata_extract "art" 0x1000 0x440
```

Comment:
No MAC address store in caldata (0x1002, 0x5006 ...)

Reply:
The MAC adresses is stored in uboot-enviroment as ASCII, just as on the Enterasys and Siemens-devices.